pkgs/servers/monitoring/facette/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub
+, go, pkgconfig, nodejs, nodePackages, pandoc, rrdtool }:
+
+stdenv.mkDerivation rec {
+  name = "facette-${version}";
+  version = "0.4.0";
+  src = fetchFromGitHub {
+    owner = "facette";
+    repo = "facette";
+    rev = "${version}";
+    sha256 = "1m7krq439qlf7b4l4bfjw0xfvjgj67w59mh8rf7c398rky04p257";
+  };
+  nativeBuildInputs = [ go pkgconfig nodejs nodePackages.npm pandoc ];
+  buildInputs = [ rrdtool ];
+  preBuild = ''
+    export HOME="$NIX_BUILD_ROOT" # npm needs a writable home
+  '';
+  installPhase = "make PREFIX=$out install";
```

Comment:
Please use  `makeFlags` instead of overwriting `installPhase`. Eg. `makeFlags = [ "PREFIX=$(out)" ];`

Reply:
Done